<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>321</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>301</width>
    <height>321</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>301</width>
    <height>321</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GUI - Hebb</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>261</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Learning Algorithm:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="choiceComboBox">
       <item>
        <property name="text">
         <string>Hebb</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Perceptron</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_train">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>250</height>
     </rect>
    </property>
    <property name="title">
     <string>Training</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_train_input">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="2" column="0">
         <widget class="QSpinBox" name="spinBox_in_3a">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QSpinBox" name="spinBox_in_3b">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSpinBox" name="spinBox_in_1b">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="spinBox_in_4a">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_in_2a">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QSpinBox" name="spinBox_in_2b">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QSpinBox" name="spinBox_in_1a">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSpinBox" name="spinBox_in_4b">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QGroupBox" name="groupBox_train_output">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSpinBox" name="spinBox_out_1">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_out_2">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QSpinBox" name="spinBox_out_3">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QSpinBox" name="spinBox_out_4">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>-1</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="trainButton">
       <property name="text">
        <string>Train</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="weightLabel_B">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;W&lt;span style=&quot; vertical-align:sub;&quot;&gt;B&lt;/span&gt;: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="weightLabel_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;W&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="weightLabel_1">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;W&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;: 0&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_run">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>50</y>
      <width>71</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Running</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_4">
     <item row="2" column="0">
      <widget class="QLabel" name="outputLabel">
       <property name="text">
        <string>Output: 0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QGroupBox" name="groupBox_run_input">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QSpinBox" name="spinBox_run_a">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QSpinBox" name="spinBox_run_b">
          <property name="minimum">
           <number>-1</number>
          </property>
          <property name="maximum">
           <number>1</number>
          </property>
          <property name="singleStep">
           <number>2</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>46</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="runButton">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Preferred</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>choiceComboBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
